src/sass/components/_timeline.scss

```diff
@@ -0,0 +1,215 @@
+   */
+
+  &__marker {
+    width: $sm*.75;
+    height: $sm*.75;
+    position: absolute;
+    color: $color-white;
+    background: $color-midnight;
+    border-radius: 50%;
+    box-shadow: 0 0 0 $xxs $color-black--100;
+    margin-top: $sm;
+  }
+
+  &__content {
+    padding: $md;
+    margin-left: $lg;
+    background-color: $color-white;
+    border-radius: $xxs;
+    box-shadow: $shadow-base;
+    border: none;
+  }
+
+  & h2 {
+    font-size: $ts-26;
+    font-weight: $font-weight-bold;
+  }
+
+  &__date {
+    font-size: $ts-14;
+    color: $color-black--700;
+  }
+}
+
+.#{$prefix}-timeline--right {
+  &::before {
+    left: unset;
+    right: $xxs;
+  }
+
+  .#{$prefix}-timeline__item::after {
+    margin-left: 0;
+    right: 0;
+    border-left-color: $color-black--100;
+    border-right-color: transparent;
+  }
+
+  .#{$prefix}-timeline__marker {
+    margin-left: 0;
+    right: 0;
+  }
+
+  .#{$prefix}-timeline__content {
+    margin-left: 0;
+    margin-right: $lg;
+  }
+}
+
+@include mq($breakpoint-md) {
+  .#{$prefix}-timeline--center {
+    &::before {
+      left: 50%;
+      margin-left: 0;
+    }
+
+    & > div {
+      display: flex;
+      flex-wrap: wrap;
+      margin-right: -$sm*.75;
+      margin-left: -$sm*.75;
+    }
+
+    .#{$prefix}-timeline__item {
+      flex-basis: 50%;
+      max-width: 50%;
+      padding: 0 $sm*.75;
+      width: 100%;
+    }
+
+    .#{$prefix}-timeline__item-right {
+      margin-left: auto;
+    }
+
+    .#{$prefix}-timeline__item::after {
+      margin-left: -$md;
+      left: 100%;
+      border-left-color: $color-black--100;
+      border-right-color: transparent;
+    }
+
+    .#{$prefix}-timeline__item-right::after {
+      margin-left: -$sm;
+      left: unset;
+      border-right-color: $color-black--100;
+      border-left-color: transparent;
+    }
+
+    .#{$prefix}-timeline__marker {
+      margin-left: -$xxs;
+      left: 100%;
+    }
+
+    .#{$prefix}-timeline__item-right .#{$prefix}-timeline__marker {
+      left: unset;
+      margin-left: -$sm;
+    }
+
+    .#{$prefix}-timeline__content {
+      margin-left: 0;
+      margin-right: $sm*.75;
+    }
+
+    .#{$prefix}-timeline__item-right .#{$prefix}-timeline__content {
+      margin-right: 0;
+      margin-left: $sm;
+    }
+
+    .#{$prefix}-timeline__date--label {
+      text-align: left;
+      position: absolute;
+      display: block;
+      left: initial;
+      right: -$sm*7;
+      top: unset;
+      margin-top: -$sm*3.15;
+    }
+
+    .#{$prefix}-timeline__item-right .#{$prefix}-timeline__date--label {
+      min-width: fit-content;
+      min-height: fit-content;
+      left: -$sm*7.5;
+      right: initial;
+      text-align: right;
+      position: absolute;
+      display: block;
+      top: $sm*.75;
+      margin-left: unset;
+      margin-top: unset;
+      margin-right: unset;
+    }
+  }
+}
+
+@include mq($breakpoint-xl) {
+  .#{$prefix}-timeline {
+    &__date--label {
+      min-width: fit-content;
```

Comment:
In order to accept longer pieces of text in the `__date` element. We'll need to set some sort of explicit width on larger screens and adjust the absolute positioning. That way if the text gets longer it will flow onto two lines. See screenshot of what currently happens:

![image](https://user-images.githubusercontent.com/5590034/61814555-50d28c00-ae16-11e9-87ce-d1baa87aaa18.png)

We should be able to fix this by changing the `min-width` property here to be something like `10rem` (that's just an arbitrary number). Once we decide on a sensible default width, we'll need to adjust the `left` position to accommodate the new fixed width of the `&__date--label` element. We can take a look at this when we meet Friday morning if you'd like. 👍